NineChronicles.Headless.Executable/Commands/TxCommand.cs

```diff
@@ -45,7 +46,11 @@ public TxCommand(IConsole console)
             string[] actions,
             [Option("bytes", new[] { 'b' },
                 Description = "Print raw bytes instead of base64.  No trailing LF appended.")]
-            bool bytes = false
+            bool bytes = false,
+            [Option("gas-limit", Description = "limit of the total amount of transaction gas fee.")]
+            long? gasLimit = null,
+            [Option("max-gas-price", Description = "maximum amount of transaction gas fee.")]
```

Comment:
I guess it means the maximum price per used tx gas. Maybe it's the awkwardness that comes from my immature English skills 😞 

```
            [Option("max-gas-price", Description = "maximum price of transaction gas fee.")]
```

Reply:
As far as I know, tx gas fee is calculated from gasLimit * maxGasPrice. gasLimit is the maximum gas allowed when calling IActionContext.UseGas in IAction.Execute, and maxGasPrice is the unit of FAV to be paid per gas.